src/compiler/checker.ts

```diff
@@ -16718,7 +16718,7 @@ namespace ts {
                     }
                 });
 
-                // ECMA-262 11.1.5 Object Initialiser
+                // ECMA-262 11.1.5 Object Initializer

```

Comment:
Is it 11.1.5?

It's not, according to the [ECMA web site](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-object-initializer)

> 12.2.6 Object Initializer
> NOTE 1 An object initializer is an expression describing the initialization of an Object, written in a form resembling a literal. It is a list of zero or more pairs of property keys and associated values, enclosed in curly brackets. The values need not be literals; they are evaluated each time the object initializer is evaluated.